scripts/deploy-upgradable.js

```diff
@@ -103,7 +103,7 @@ async function deploy(env, chains, wallet, artifactPath, contractName, deployTo)
                 implementationJson,
                 args,
                 getUpgradeArgs(contractName, chain),
-                chain.gasOptions,
+                _.get('gasOptions.gasLimit', chain),
```

Comment:
Ah, I see. Should refactor this in future to take all gas options (might need to override gas price, gas tip etc.).